content/docs/concepts/remote_write_spec_2_0.md

```diff
@@ -0,0 +1,394 @@
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Introduction
+
+### Background
+
+The remote write protocol is designed to make it possible to reliably propagate samples in real-time from a sender to a receiver, without loss.
+
+The remote write protocol is designed to make stateless implementations of the server possible; as such there are little-to-no inter-message references.  As such the protocol is not considered "streaming." To achieve a streaming effect multiple messages should be sent over the same connection using e.g. HTTP/1.1 or HTTP/2. "Fancy" technologies such as gRPC were considered, but at the time were not widely adopted, and it was challenging to expose gRPC services to the internet behind load balancers such as an AWS EC2 ELB.
+
+The remote write protocol contains opportunities for batching, e.g. sending multiple samples for different series in a single request. It is not expected that multiple samples for the same series will be commonly sent in the same request, although there is support for this in the protocol.
+
+<!---
+TODO(bwplotka): Challenge last sentence (e.g. OTel coll usage)?
+-->
+The remote write protocol is not intended for use by applications to push metrics to Prometheus remote-write-compatible Receiver. It is intended that a Prometheus remote-write-compatible sender scrapes instrumented applications or exporters and sends remote write messages to a server.
+
+<!---
+TODO(bwplotka): Add support for 2.0 to those suites 
+-->
+A test suite can be found at https://github.com/prometheus/compliance/tree/main/remote_write_sender.
+
+### Glossary
+
+For the purposes of this document the following definitions MUST be followed:
+
+* a "Sender" is something that sends Prometheus Remote Write data.
+* a "Receiver" is something that receives Prometheus Remote Write data.
+* a "Sample" is a pair of (timestamp, value).
+* a "Histogram" is a pair of (timestamp, histogram value).
+* a "Label" is a pair of (key, value).
+* a "Series" is a list of samples, identified by a unique set of labels.
+
+## Definitions
+
+### Protocol
+
+The Remote Write Protocol MUST consist of RPCs with the request body encoded using a Google Protobuf 3 message. The protobuf encoding MUST use either of the following schemas: 
+
+<!---
+TODO(bwplotka): Do we deprecate it?
+-->
+* [`prometheus.WriteRequest`](./remote_write_spec.md#protocol) introduced in the Remote Write 1.0 specification. As of 2.0 the `prometheus.WriteRequest` message is deprecated.
+* `io.prometheus.write.v2.Request` introduced in this specification and defined [below](#ioprometheuswritev2request-proto-schema). Senders SHOULD use `io.prometheus.write.v2.Request` when possible.
```

Comment:
:+1: if we're keeping info about the 2.0 proto format then I like this section `protobuf encoding MUST use either of the following schemas:`